server/app/controllers/api/v1/flights_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::FlightsController < ApiController
+  def create
+    if flight_info_params[:type] == "one_way"
+      flights = search_one_way_flight
+
+      @flights = flights
+      render_response @flights.size.positive?
+
+    elsif flight_info_params[:type] == "round_trip"
+      first_flights, second_flights = search_round_trip_flight
+
+      @first_flights = first_flights
+      @second_flights = second_flights
+      render_response @first_flights.size.positive? || @second_flights.size.positive?
+    end
+  end
+
+  private
+
+  def flight_info_params
+    params.require(:flight).permit(
+      :type,
+      :ticket_number,
+      time: [:first, :second],
+      locations: [:from, :to]
+    )
+  end
+
+  def search_one_way_flight
+    SearchOneWayFlightService.new(flight_info_params).perform
+  end
+
+  def search_round_trip_flight
+    SearchRoundTripFlightService.new(flight_info_params).perform
+  end
+
+  def render_response condition
```

Comment:
render :create if condition
render json: {success: false, message: "Flights not found !!"}

Reply:
Để như trên em bị dính lỗi ạ
![Screen Shot 2020-08-18 at 10 05 32 AM](https://user-images.githubusercontent.com/68627414/90465841-6fe73e80-e13a-11ea-9faf-56c3447f7d9e.png)